spec/flows/steps/helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Steps::Helper do
+  describe "#relevant_step?" do
+    context "when there is asylum support data plus income data" do
```

Comment:
question - do we need to specify this is a certificated check in the spec and doe we want a separate one for controlled?

Reply:
hmm possibly - I might add a few more specs to beef this file up 👍